.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug Report
+about: Report a bug encountered with the gateway-api project
```

Comment:
let's tailor this for kube-agentic-networking project.
There's a few places where Gateway API is mentioned in the template